src/config/dynamic-theme-fixes.config

```diff
@@ -13725,6 +13725,15 @@ g#ubuntu-logo > path
 
 udemy.com
 
+CSS
+.radio:not(.radio-button) input[type="radio"]:checked + .radio-label::before, .radio-inline:not(.radio-button) input[type="radio"]:checked + .radio-label::before {
+    background-color: rgb(93, 122, 136);
+    box-shadow: rgb(27, 29, 29) 0px 0px 0px 2px inset;
```

Comment:
Is the box-shadow necessary to be modified?

Reply:
it's not necessary, I was trying to make it smoother

Reply:
I consider it as a cosmetic change and not necessary a fix.